public/locales/pt/common.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "O modo mais fácil de criar um",
+  "readme": "README",
+  "description": "Nosso simples editor permite que você facilmente adicione e customize todas as seções que você precisa para o readme do seu projeto",
+  "get-started": "Comece aqui",
+  "100-free": "100% Gratuito",
+  "made-with-love": "Feito com ",
+  "by": " por",
+  "language": "Idioma",
```

Comment:
`Language` in portugese on google translate shows up as `língua`. Is this correct?

Reply:
It can be `Língua` too, but I think `Idioma` is more friendly, discord for example uses `Idioma` instead of `Língua` haha